R/stat-qq-line.R

```diff
@@ -0,0 +1,131 @@
+#' A quantile-quantile line
```

Comment:
I took a crack at combining them, but I'm not sure if the results are satisfactory. I don't have much experience with roxygen.